### PR TITLE
bugfix/#5641,#5649,#5598,#5599,#5705,#5717,#5718-Some-UI-fixes

### PR DESCRIPTION
**Pull request by NadiiaStoiko**
#5641
Before:
The 'Save changes' button is enabled when user navigates from the 'User data' page to the editable 'User data' page.
After:
The 'Save changes' button disable when user navigates from the 'User data' page to the editable 'User data' page.
#5649,
Before:
The Tarrif`s name and description are displayed in Ukrainian for the EN site language.
After:
The Tarrif`s name and description are displayed in English for the EN site language.
#5493,
Before:
After:
#5598,
Before:
The content of the error message under the 'Password' field at 'Sign-in form page' is incorrect.
After:
The content of the error message under the 'Password' field at 'Sign-in form page' ccording to mockup
#5599,
Before:
The 'Fonts' at the 'Event' page didn't match with mockup fonts. 
After:
The 'Fonts' at the 'Event' page according to mockup fonts.
#5705,
Before: 
Admin cannot enter 1 to 4 allowed characters in the field “Свій варіант” at cancel order  pop-up window. 
Admin can enter up to 255 characters in the field “Свій варіант” when adds a reason to cancel the order on the order form.
After:
#5717
Before:
Add reason to cancel order  pop-up window closes after clicking on the 'Add' button when admin doesn't select the reason to cancel the order.
After:
'Add' button should be disabled if admin doesn't select the reason to cancel the order.
Without task:
Removed loader when filtration of orders on order page was finished

### CODE REVIEW

**Review by KKatiaa — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts [hunk 1]
```diff
@@ -43,8 +46,19 @@ export class UbsAdminTariffsAddServicePopUpComponent implements OnInit, OnDestro
...
+    this.setDate();
+  }
+
+  setDate(): void {
+    const lang = this.languageService.getCurrentLanguage();
+    this.datePipe = new DatePipe(lang);
+    this.newDate = this.datePipe.transform(new Date(), 'MMM dd, yyyy');
+    if (lang === 'en') {
+      this.isLangEn = true;
+    }
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts]: this.isLangEn = lang === 'en'

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-tariff-service-pop-up/ubs-admin-tariffs-add-tariff-service-pop-up.component.ts [hunk 2]
```diff
@@ -44,10 +47,16 @@ export class UbsAdminTariffsAddTariffServicePopUpComponent implements OnInit {
...
+    this.setDate();
+  }
+
+  setDate(): void {
+    const lang = this.languageService.getCurrentLanguage();
+    this.datePipe = new DatePipe(lang);
+    this.newDate = this.datePipe.transform(new Date(), 'MMM dd, yyyy');
+    if (lang === 'en') {
+      this.isLangEn = true;
+    }
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-tariff-service-pop-up/ubs-admin-tariffs-add-tariff-service-pop-up.component.ts]: the same.   setDate - duplicate method, move it to the service

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts (hunk 1 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.ts]: Thank you. Fixes was added.

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-tariff-service-pop-up/ubs-admin-tariffs-add-tariff-service-pop-up.component.ts (hunk 2 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-tariff-service-pop-up/ubs-admin-tariffs-add-tariff-service-pop-up.component.ts]: Thank you. Fixes was added. Created method in to the service.

**Review by hnativlyubomyr — APPROVED** [review]

**Review by KKatiaa — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by ruslan-teh — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

**Pull request merged**